### PR TITLE
Path: Fix help text for vcarve filter control

### PR DESCRIPTION
**Pull request by sliptonic**
Thank you for creating a pull request to contribute to FreeCAD! To ease integration, please confirm the following:

- [ ] Branch rebased on latest master `git pull --rebase upstream master`
- [ ] Unit tests confirmed to pass by running `./bin/FreeCAD --run-test 0`
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)
- [ ] Commit message includes `issue #<id>` or `fixes #<id>` where `<id>` is the [associated MantisBT](https://freecadweb.org/wiki/tracker#GitHub_and_MantisBT) issue id if one exists

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

### CODE REVIEW

File: src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui [hunk 1]
```diff
@@ -95,14 +95,14 @@
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
-         <string>Threshold</string>
+         <string>Filter Colinear lines</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QSpinBox" name="threshold">
```
Comment (chennes) [inline src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui:103]: Do you want to leave the spinbox's name as "threshold" here?

File: src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui [hunk 2]
```diff
@@ -95,14 +95,14 @@
...
         <property name="toolTip">
-         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Threshold is used by the medial axis filter to remove unwanted segments. If the resulting path contains unwanted segments, decrease this value. &lt;/p&gt;&lt;p&gt;Valid values are 0.0 - 1.0&lt;/p&gt;&lt;p&gt;Default = 0.8&lt;/p&gt;&lt;p&gt;1.0 will remove nothing.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
+         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Controls how aggressively colinear segments are filtered from the voronoi diagram&lt;/p&gt;&lt;p&gt;Valid values are 0 - 90 degrees (larger numbers filter more)&lt;/p&gt;&lt;p&gt;Default = 10&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="maximum">
          <number>180</number>
```
Comment (chennes) [inline src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui:108]: Your tooltip says the max is 90.

**Review by chennes — COMMENTED** [review]
A few comments placed inline in the XML, hopefully they make sense as you are looking at it in Designer:

File: src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui (hunk 1 above)
Comment (sliptonic) [reply src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui:103]: Not gonna let me get away with being lazy, are ya?    Fine.  Fixed.

File: src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui (hunk 2 above)
Comment (sliptonic) [reply src/Mod/Path/Gui/Resources/panels/PageOpVcarveEdit.ui:108]: fixed.  Thanks.

**Comment by sliptonic** [comment]
Note that changing the property names from Threshold to Colinear will break existing operations.  They  will have to be recreated.

**Pull request merged**